plugins/modules/vyos_logging_global.py

```diff
@@ -715,17 +721,17 @@
   returned: when I(state) is C(merged), C(replaced), C(overridden), C(deleted) or C(purged)
   type: list
   sample:
-    - "set system syslog console facility local7 level err"
-    - "set system syslog host 172.16.0.1 port 223"
-    - "set system syslog global archive size 111"
+    - sample command 1
```

Comment:
Does "sample command" need to be replaced by a real one?

Reply:
yes, thanks.